test/manipulation.test.js

```diff
@@ -1024,6 +1024,18 @@ describe('manipulation', function() {
       .catch(done);
       });
 
+      it('should fail when changing id', function(done) {
+        Post.findById(postInstance.id, function(err, p) {
+          if (err) return done(err);
+          p.replaceAttributes({ title: 'b', id: 999 }, function(err, p) {
+            should.exist(err);
+            var errMsg = 'id property (id) cannot be updated from ' + postInstance.id + ' to 999';
```

Comment:
Please include "expected" in the variable name, e.g. `expectedErrMsg` or `expectedError`